benchmarks/benchmark/tools/profile-generator/container/benchmark_serving.py

```diff
@@ -545,5 +607,11 @@ def main(args: argparse.Namespace):
           " the form of a string."
       ),
   )
+  parser.add_argument(
+      "--scrape-server-metrics",
+      type=bool,
```

Comment:
For other arguments that are booleans like this we are using `action=store_true` like `--save-json-result` or `--use-dummy-text` where specifying just the flag is enough. But it seems like setting it to a bool allows us to easily specify values at runtime. Maybe we can switch the others to bool as well if we want to follow the same pattern. Not a blocker. But something to address.

Reply:
I double checked to see if using the type bool worked with false - turns out python ignored what was set if the flag was passed in. I was able to fix this doing something similar to what we have in the locust run.sh file.

Ref: https://github.com/GoogleCloudPlatform/ai-on-gke/blob/0294ba355e8d0809fc4c37e7dcd0365358521af4/benchmarks/benchmark/tools/locust-load-inference/locust-docker/locust-tasks/run.sh#L19